tier4_planning_msgs/msg/PlanningFactor.msg

```diff
@@ -0,0 +1,24 @@
+# msg format version
+string version_msg_format
```

Comment:
How do you plan to use the version? With ROS messages, there is no flexibility in the type, so I don't think you can make use of it very effectively.

Reply:
It is removed.
https://github.com/tier4/tier4_autoware_msgs/pull/155/commits/ca8a02461c74d374f1e02a55fe8350d723178efb